Makefile

```diff
@@ -48,8 +48,6 @@ update_dependencies:
 	make update_starscream version=4.0.4
 	echo "👉 Updating SwiftyGif"
 	make update_swiftygif version=5.4.2
-	echo "👉 Updating SwiftyMarkdown"
-	make update_swiftymarkdown version=master
```

Comment:
don't think this is removing the dependency, it's only preventing updates

Reply:
Ah I see..

Reply:
We did a similar approach on DiffKit, where we updated the source code, and made sure that part is not updated as far as I remember right? @laevandus

Reply:
Here: https://github.com/GetStream/stream-chat-swift/pull/2996/files#r1475713507

We probably should use the same approach to have consistency

Reply:
Yeah, I modified the script not to overwrite a file

Reply:
Updated 👍

But instead of removing the file as we do with DiffKit, I implemented a `git restore` function to always keep the existing version of `SwiftyLineProcessor.swift`